test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -58,6 +65,13 @@ def test_successful_purchase
     assert_equal 'Succeeded', response.message
   end
 
+  def test_successful_purchase_with_network_token
```

Comment:
Nice! Could you also add one to the non-remote unit tests for consistency sake. The stubbed responses there sometimes act as a useful reference.